docs/cep/getting-started-stream-workers.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 10
+title: Getting Started with Stream Workers
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page explains how to create a stream worker that reads data added to a collection and sends the data to a stream. After you create the stream worker, there are instructions for testing it. 
+
+## Create a Stream Worker
+
+Refer to the [Stream Workers](/cep/index.md) section for an explanation of how stream workers function.
+
+1. [Log in to your Macrometa account](https://auth-play.macrometa.io/).
+1. Click **Stream Workers**.
+1. In the Editor tab, define the stream worker. Copy and paste the following code block in the code editor on the Editor tab.
+
+```sql
+@App:name('sample-cargo-app')
+@App:description('Basic stream worker to demonstrate reading data from a collection and sending it to a stream. The stream and collection are created automatically if they do not already exist.')
+@App:qlVersion('2')
+
+/**
+Testing the Stream Worker:
+    1. Open Stream `SampleCargoAppDestStream` in Console. The output can be monitored here.
+
+    2. Upload following data into `SampleCargoAppInputTable` C8DB Collection.
+        {"weight": 1}
+        {"weight": 2}
+        {"weight": 3}
+        {"weight": 4}
+        {"weight": 5}
+
+    3. Following messages would be shown on the `SampleCargoAppDestStream` Stream Console.
+        [2021-08-27T14:12:15.795Z] {"weight":1}
+        [2021-08-27T14:12:15.799Z] {"weight":2}
+        [2021-08-27T14:12:15.805Z] {"weight":3}
+        [2021-08-27T14:12:15.809Z] {"weight":4}
+        [2021-08-27T14:12:15.814Z] {"weight":5}
+*/
+
+-- Defines `SampleCargoAppInputTable` Source.
+CREATE SOURCE SampleCargoAppInputTable WITH (type = 'database', collection = "SampleCargoAppInputTable", collection.type="doc" , replication.type="global", map.type='json') (weight int);
+
+-- Define `SampleCargoAppDestStream` Stream.
+CREATE SINK STREAM SampleCargoAppDestStream (weight int);
+
+-- Data Processing
+@info(name='Query')
+INSERT INTO SampleCargoAppDestStream
+SELECT weight
+FROM SampleCargoAppInputTable;
+```
+
+1. Click **Validate**. Macrometa checks to see that your code is valid.
+1. Click **Save**.
+1. Select edge locations, and then click **Save**. The locations that you select represent where the data for this stream worker will live.
+1. Click **Publish** to publish your stream worker. When you do this, it will begin to run the code as defined and will continue until you unpublish it.
+
+## Test the Stream Worker
+
+Use the following procedure to verify that the stream worker functions as expected.
+
+### 1. Open a Stream Window
+
+The Macrometa Streams console does not persist messages, so to see them, you must have the console open before you send.
```

Comment:
Change to: Because the Macrometa streams console does not persist messages, you'll want to have the console open in two browser tabs.